go-controller/pkg/clustermanager/egressservice/egressservice_cluster_service.go

```diff
@@ -118,7 +118,7 @@ func (c *Controller) onServiceDelete(obj interface{}) {
 // When all the endpoints of the given service are host-networked,
 // the function returns an empty slice as there is no point of allocating such service.
 func (c *Controller) backendNodesFor(svc *corev1.Service) ([]string, error) {
-	endpointSlices, err := c.watchFactory.GetEndpointSlices(svc.Namespace, svc.Name)
+	endpointSlices, err := c.watchFactory.GetEndpointSlices(svc.Namespace, svc.Name, "")
```

Comment:
would it be better to substitute "" with some constant so that someone who makes the controllers network aware later can just search for that constant and know where they need to update it to select the correct network? like TempNotNetworkAware or something

Reply:
To be more explicit I modified the calls/function to use the default network name.
Like with EgressServices I would prefer to have the TODOs at a top level, in a place where we start the controller.